src/foam/u2/view/UnstyledTableView.js

```diff
@@ -390,8 +390,8 @@ foam.CLASS({
 
                   if ( checked ) {
                     view.selectedObjects = {};
-                    view.data.select(function(obj) {
-                      view.selectedObjects[obj.id] = obj;
+                    view.data.inX(ctrl.__subContext__).select().then(function(obj) {
```

Comment:
What is this for? This breaks the use of the UnstyledTableView by FOAM apps that do not use NANOS and don't have 'ctrl'.

Reply:
The select didn't work unless we added the inX. In most cases it would just return an empty dao.